crates/matrix-sdk/src/client/builder.rs

```diff
@@ -107,6 +109,9 @@ impl ClientBuilder {
             requires_sliding_sync: false,
             #[cfg(feature = "experimental-sliding-sync")]
             sliding_sync_proxy: None,
+            // Simplified MSC3575 is turned on by default for the SDK.
+            #[cfg(feature = "experimental-sliding-sync")]
+            is_simplified_sliding_sync_enabled: true,
```

Comment:
Hmm, what happens if the server doesn't support triple S?

Reply:
The SDK has to be configured to choose between SS or SSS. I believe the same question holds if the server doesn't support the SS proxy? If we decide to choose `false` for the default option, it means we have to update the tests in the opposite direction (when mocking the response from the server).

Reply:
Alright, feel free to leave it as is.